components/electric/lib/electric/satellite/serialization.ex

```diff
@@ -621,36 +633,68 @@ defmodule Electric.Satellite.Serialization do
     _ = Time.from_iso8601!(val)
 
     val
+  rescue
+    _ -> raise DataValidationError, message: "Unexpected value for :time column: #{inspect(val)}"
```

Comment:
The same considerations as for the other use of `rescue` above apply here 

```suggestion
    _ -> reraise DataValidationError, [message: "Unexpected value for :time column: #{inspect(val)}"], __STACKTRACE_
```